.travis.yml

```diff
@@ -1,20 +1,36 @@
 language: go
 sudo: required
 services:
-  - docker
+- docker
```

Comment:
It's really Nitpicking but even if we can declare list element on the same indentation than the list name, the recommended syntax is to indent them http://yaml.org/start.html  
Plus I find it more readable :nerd_face: